src/internal/base-component/metrics/interfaces.ts

```diff
@@ -27,6 +27,7 @@ export interface ComponentMetricDetail {
   // "used" – individual component used
   action: 'loaded' | 'used';
   configuration?: ComponentConfiguration;
+  packageSource?: string;
```

Comment:
Why is it called `packageSource` instead of `sourcePackage` or just `package`?

Reply:
Based on the existing `context.packageSource`, but I don't have a strong preference.

Could also be `packageName` for consistency with `componentName`.

Let's call it `package` for simplicity?